spec.md

```diff
@@ -2524,94 +2484,6 @@ circuit Foo: %[[
 Any legal JSON is allowed, meaning that the above JSON may be stored "minimized"
 all on one line.
 
-# The Lowered FIRRTL Forms
```

Comment:
Maybe we could change this section to just define LoFirrtl without any requirement that compilers must implement it. Such a definition would be convenient as some tools like simulators or formal backends might want to refer to LoFirrtl. One example might be that CIRCT could be used to convert to LoFirrtl, desugaring any new frontend features, and then that LoFirrtl could be fed into the existing SMT backend which does not care about high level features. Of course, there should not be any requirement that a compliant compiler always goes through LoFirrtl.